python/tests/scripts/c3_pln_stv.metta

```diff
@@ -27,24 +37,36 @@
    )
 )
 
-
-; Reproducing examples/pln/conjunction (extended with implication)
```

Comment:
Why is concrete reference to examples/pln/conjunction is dropped?

Reply:
my bad, restored this info in the file header